Dockerfile

```diff
@@ -0,0 +1,65 @@
+## to build:
+#
+# docker build -t halprin/fcli .
+#
+## to run:
+#
+# docker run -v ~/.fcli:/.fcli halprin/fcli fcli [...]
+#
+# -v adds a mapping so that your ~/.fcli is accessible in the container
+# halprin/fcli is the image to run
+# fcli is the command to run
+# [...] are the arguments to the fcli commandFROM alpine:latest
+
+# use Alpine Linux -- the resulting image is ~210mb
+FROM alpine:latest
```

Comment:
There is already the easy to use image `python:3-alpine`.  https://hub.docker.com/_/python/
Check the `Create a Dockerfile in your Python app project` section at that link above to see a way that could easily be adapted to run `fcli`.